Apps/CocoaPods-FCM/src/View/CustomAttributeView.swift

```diff
@@ -0,0 +1,77 @@
+import CioTracking
+import SwiftUI
+
+struct CustomAttributeView: View {
```

Comment:
- What did you gain by having a conditional view?
- Could you have made it even MORE conditional and not hardcoded so much? It seems like if the View knew which type of Attribute it was tracking that it could have a single String for everything that just replaces a param in each with the Attribute type when it came to it. 
Something like

```
    let attributeType: AttributeType
    
    private var headerText: String
    private var buttonText: String

    init(attributeType: String) {
        self.headerText = "Set Custom \(attributeType) Attribute"
        self.buttonText = "Send \(attributeType) Attributes"
    }
```